db/seeds.rb

```diff
@@ -1,42 +1,42 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
-#
-# Examples:
-#
-#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
+Jobtag.destroy_all
+ActiveRecord::Base.connection.reset_pk_sequence!("jobtags")
+Tag.destroy_all
+ActiveRecord::Base.connection.reset_pk_sequence!("tags")
+Job.destroy_all
+ActiveRecord::Base.connection.reset_pk_sequence!("jobs")
+Company.destroy_all
+ActiveRecord::Base.connection.reset_pk_sequence!("companies")
+
 30.times do
   Job.create(
-              position: Faker::Name.title,
-              posting_date: rand(1...10).days.ago,
-              description: Faker::Lorem.paragraphs(3),
-              location: "#{Faker::Address.city}, #{Faker::Address.state}",
-              company_id: rand(1...10),
-              source: Faker::Name.name
-            )
+    position: Faker::Name.title,
+    posting_date: rand(1...10).days.ago,
+    description: Faker::Lorem.paragraphs(3),
+    location: "#{Faker::Address.city}, #{Faker::Address.state}",
+    company_id: rand(1...10),
+    source: Faker::Name.name
+  )
 end
 puts "Jobs Created"
 
 10.times do
   Company.create(
-                 name: "#{Faker::Company.name} #{Faker::Company.suffix}",
-                 url: Faker::Internet.url,
-                 location: "#{Faker::Address.city}, #{Faker::Address.state}",
-                )
+    name: "Apple",
```

Comment:
Had to give a legit company name so glassdoor could populate the rest of the info